app/views/player.py

```diff
@@ -140,6 +141,58 @@ def get(self, request):
         })
 
 
+@method_decorator(running_game_required, name='dispatch')
+@method_decorator(player_required, name='dispatch')
+class MessagePlayersView(View):
+    template_name = 'dashboard/message_players.html'
+
+    def get(self, request, **kwargs):
+        game = most_recent_game()
+        participant = request.user.participant(game)
+        message_players_form = kwargs.get('message_players_form', MessagePlayersForm(player=participant))
+
+        return render(request, self.template_name, {
+            'game': game,
+            'participant': participant,
+            'message_players_form': message_players_form
+        })
+
+    def post(self, request):
+        game = most_recent_game()
+        participant = request.user.participant(game)
+        message_players_form = MessagePlayersForm(request.POST, player=participant)
+        if not message_players_form.is_valid():
+            return self.get(request, message_players_form=message_players_form)
+
+        cd = message_players_form.cleaned_data
```

Comment:
unclear variable name tbh, don't we use "cleaned_data" or some other variations in other places